mne_qt_browser/_pg_figure.py

```diff
@@ -69,6 +69,29 @@ def _get_std_icon(icon_name):
         getattr(QStyle, icon_name))
 
 
+def _get_color(color_spec):
+    """Wraps mkColor to accept all possible matplotlib color-specifiers."""
+    try:
+        # Convert matplotlib color-names if possible
+        color_spec = _to_rgb(color_spec, alpha=True)
+    except ValueError:
+        pass
+
+    # Convert tuples of floats from 0-1 to 0-255 for pyqtgraph
+    if (isinstance(color_spec, tuple) and
+            all([i <= 1 for i in color_spec])):
+        color_spec = tuple([int(i * 255) for i in color_spec])
+
+    try:
+        color = mkColor(color_spec)
+    except ValueError:
+        logger.critical(f'{color_spec} is not a valid matplotlib '
+                        f'color-specifier!')
+        return mkColor((0, 0, 0, 128))
```

Comment:
I think this should emit an error like
```
raise ValueError(...) from None
```

Reply:
Ok, so the plot will fail then

Reply:
Yeah I think if you give an invalid matplotilb color you should get an error and the plot should fail